index.js

```diff
@@ -13,9 +13,12 @@ const config = require('./config');
 config.init();
 const plugins = require('./plugins');
 
+const windowSet = new Set([]);
+
 // expose to plugins
 app.config = config;
 app.plugins = plugins;
+app.getWindows = () => [...windowSet];
```

Comment:
A new `Set` sounds good. I think if we internally use a `Set`, it makes sense to expose a `Set`. Good observation. Thank you